src/rules/tsrDetectPossibleTimingAttacksRule.ts

```diff
@@ -84,6 +84,7 @@ export class Rule extends Lint.Rules.AbstractRule {
 
 class RuleWalker extends Lint.RuleWalker {
     visitBinaryExpression (node: ts.BinaryExpression) {
+        const operatorTokenKind = node.operatorToken.kind;
```

Comment:
Great! But below all `node.operatorToken.kind` expressions should be replaced with `operatorTokenKind` ;)

Reply:
This is embarrassing... forgot to hit ctrl+s :D